aws/templates/solution/solution_ecs.ftl

```diff
@@ -53,6 +53,47 @@
             name=formatComponentLogGroupName(tier, component) /]
     [/#if]
 
+    [#if deploymentSubsetRequired("efs", true) && 
+            ecs.ClusterWideStorage?has_content && ecs.ClusterWideStorage ]
+
+        [#assign ecsEFSVolumeId = formatEFSId(tier, component)]
+        [#assign ecsEFSVolumeName = formatName( tier, component, "efs")]
+        [#assign ecsEFSSecurityGroupId = formatComponentSecurityGroupId( tier, component,"efs")]
+        [#assign ecsEFSIngressSecurityGroupId = formatDependentSecurityGroupIngressId(ecsEFSSecurityGroupId) ]
+
+        [@createComponentSecurityGroup
+            mode=listMode
+            tier=tier
+            component=component 
+            extensions="efs"
+            /]
+        
+        [@createSecurityGroupIngress
+            mode=listMode
+            id=ecsEFSIngressSecurityGroupId
+            port="any"
+            cidr=ecsSecurityGroupId
+            groupId=ecsEFSSecurityGroupId
+        /]
+
+        [@createEFS 
+            mode=listMode
+            tier=tier
+            id=ecsEFSVolumeId
+            name=ecsEFSVolumeName
+            component=component
+        /]
+
+        [@createEFSMountTarget
+            mode=listMode
+            tier=tier
+            efsId=ecsEFSVolumeId
+            securityGroups=ecsEFSSecurityGroupId
+            dependencies=[ecsEFSVolumeId,ecsEFSSecurityGroupId]
```

Comment:
do we explicitly need dependencies given both are parameters to macro anyway so CF will likely figure them out for itself...

Reply:
Removed